File: src/main/java/com/example/demo/entity/Orderr.java
package com.example.demo.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
public class Orderr {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int idorder;
	private int numtable;
	private int idreservetable;
	private String drinks;
	private Double total;
	private boolean paid;

}

File: lib/models/orderr.dart
class Orderr {
  int idorder;
  int numtable;
  int idreservetable;
  String drinks;
  double total;
  bool paid;

  Orderr({
    required this.idorder,
    required this.numtable,
    required this.idreservetable,
    required this.drinks,
    required this.total,
    required this.paid,
  });

  factory Orderr.fromJson(Map<String, dynamic> json) {
    return Orderr(
      idorder: json['idorder'] as int,
      numtable: json['numtable'] as int,
      idreservetable: json['idreservetable'] as int,
      drinks: json['drinks'] as String,
      total: (json['total'] as num).toDouble(),
      paid: json['paid'] as bool,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'idorder': idorder,
      'numtable': numtable,
      'idreservetable': idreservetable,
      'drinks': drinks,
      'total': total,
      'paid': paid,
    };
  }
}
